proposals/datetime-picker-tools.md

```diff
@@ -0,0 +1,147 @@
+# baseui/datepicker
+
+This is a proposal for enhancements to the existing collection of datepicker components.
+
+## Motivation
+
+Date/time selections need more precision than a single date or range of dates (what is currently implemented). This proposal outlines two new components and changes to one existing component. `TimePicker`, `TimezonePicker`, and `DatePicker`.
+
+## APIs and Components
+
+`TimePicker` - used to select a time, decoupled from a date. E.g. `11:50AM`.
+
+API:
```

Comment:
I think the most of world uses 24h time format. It would be nice to have a mode for that? Or even having some mapping function to create your own format (but 12h vs 24h might be good enough).

Reply:
added a 12/24 format